addons/hr_holidays/models/hr_leave.py

```diff
@@ -362,7 +362,7 @@ def _onchange_employee_id(self):
     @api.onchange('date_from', 'date_to', 'employee_id')
     def _onchange_leave_dates(self):
         if self.date_from and self.date_to:
-            self.number_of_days = self._get_number_of_days(self.date_from, self.date_to, self.employee_id.id)
+            self.number_of_days = self._get_number_of_days(self.date_from, self.date_to, self.employee_id.id)['days']
```

Comment:
This can potentially crash if `self.employee_id.id` is False
Because in this case `_get_number_of_days` will send you a tuple `(days, hours)`